amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -60,8 +65,13 @@
                                         selected, buffers the input data in memory before reading and writes the output
                                         data to an in-memory buffer instead of a file. [default: file]
 
+    -P --benchmark-result-previous <file_path>      This option will specify the path of benchmark result from the
+    existing ion-python commit.
+
+    -X --benchmark-result-new <file_path>           This option will specify the path of benchmark result form the new
```

Comment:
form -> from

Reply:
Added. Also changed a same typo in ion-java-benchmark-cli in my next PR for formats conversion.